src/components/Comments/ContainerComment.js

```diff
@@ -138,12 +260,11 @@ const ContainerComment = ({
                       isLast={isLast}
                       isLastInParent={isLastInParent}
                       comments={comments}
-                      navigateToReplyView={navigateToReplyView}
                       findCommentAndUpdate={findCommentAndUpdate}
                       hideLeftConnector={hideLeftConnector}
-                      navigation={navigation}
                       updateVote={handleUpdateVote}
                       feedId={feedId}
+                      onReplyButtonClick={onReplyButtonClick}
                     />
                   ) : null}
                 </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [250-267]

The removal of the `navigateToReplyView` prop from the `ListComment` component could potentially impact functionality if it was being used. Ensure that this change doesn't break any existing features or functionality.



<!-- This is an auto-generated comment by OSS CodeRabbit -->